sqlmesh/core/audit/definition.py

```diff
@@ -150,6 +151,10 @@ def __str__(self) -> str:
         path = f": {self._path.name}" if self._path else ""
         return f"{self.__class__.__name__}<{self.name}{path}>"
 
+    @property
+    def dbt_node_info(self) -> t.Optional[DbtNodeInfo]:
+        return self.dbt_node_info_
```

Comment:
Looks like the property method doesn't add anything extra. Why not use `dbt_node_info` as the attribute name directly?